### PR TITLE
Story 1

### PR DESCRIPTION
**Pull request by henrymeager**
Completed story 1 displaying all of my data on my homepage and added some basic styling to make it more readable.

### CODE REVIEW

File: src/VideogamesViewHelper.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?php
+
+require_once 'src/Videogames.php';
+/**
+ * The job of a view helper is contain display functionality
+ * for a given 'thing'
+ */
+class VideogamesViewHelper
```
[the remark this answers is not in the document]
Comment (ashleycoles) [reply src/VideogamesViewHelper.php:8]: Before you write them, make sure that you're definitely happy with the HTML they return

File: src/VideogamesModel.php [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        // And plugging the data into a Product entity object
+        $videogameObj = new Videogames(
+            $videogames['name'], 
+            $videogames['id'], 
+            $videogames['release_year'],
+            $videogames['platform_id'],
+            $videogames['platform_name']
+        );
+        return $videogameObj;
+    }
```
[the remark this answers is not in the document]
Comment (ashleycoles) [reply src/VideogamesModel.php:36]: If you think you will need it in the future, you can keep it though - just make sure you rename it accordingly

**Review by ashleycoles — CHANGES_REQUESTED** [review]
Make sure you remove all the extra product comments :P

File: src/VideogamesModel.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (henrymeager) [reply src/VideogamesModel.php:36]: I've removed it

**Pull request merged**